app/src/main/java/com/imashnake/animite/features/home/Home.kt

```diff
@@ -48,94 +48,108 @@ fun Home(
 
     when {
         trendingNowMediaList != null && popularThisSeasonMediaList != null -> {
-            Box(
-                modifier = Modifier
-                    .verticalScroll(rememberScrollState())
-                    .navigationBarsPadding()
-            ) {
-                Box {
-                    Image(
-                        painter = painterResource(id = Res.drawable.background),
-                        contentDescription = "Background",
-                        modifier = Modifier.fillMaxWidth()
-                    )
-
-                    Text(
-                        text = "おかえり",
-                        color = Text,
-                        fontSize = 57.sp,
-                        modifier = Modifier
-                            .align(Alignment.BottomStart)
-                            .padding(start = 24.dp, bottom = (24 + 18).dp)
-                    )
-                }
-
-                // TODO: Use `verticalArrangement` instead of the `Spacer`s.
-                Column {
-                    Spacer(
-                        modifier = Modifier.size((LocalConfiguration.current.screenWidthDp - 18).dp)
-                    )
-
-                    Column(
-                        modifier = Modifier
-                            .clip(backdropShape)
-                            .background(Backdrop)
-                    ) {
-                        Spacer(modifier = Modifier.size(24.dp))
-
-                        Text(
-                            text = "Trending Now",
-                            color = Text,
-                            fontSize = 14.sp,
-                            fontFamily = manropeFamily,
-                            fontWeight = FontWeight.Bold,
-                            modifier = Modifier.padding(start = 24.dp)
+            AnimiteTheme {
+                Box(
+                    modifier = Modifier
+                        .verticalScroll(rememberScrollState())
+                        .navigationBarsPadding()
+                ) {
+                    Box {
+                        Image(
+                            painter = painterResource(id = Res.drawable.background),
+                            contentDescription = "Background",
+                            modifier = Modifier.fillMaxWidth()
                         )
 
-                        Spacer(modifier = Modifier.size(12.dp))
-
-                        MediaSmallRow(
-                            mediaList = trendingNowMediaList,
-                            onItemClick = { itemId ->
-                                navigator.navigate(
-                                    MediaPageDestination(
-                                        id = itemId,
-                                        mediaTypeArg = homeMediaType.rawValue
+                        Box(
+                            modifier = Modifier
+                                .background(
```

Comment:
Surely this background can go directly on the parent Composable

Reply:
But then it would also appear on top of the `Text` no? I want it to be sandwiched between the `Image` and `Text`. :thinking:

Reply:
```kt
@Preview
@Composable
fun Sandwich() {
    Box {
        Box(Modifier.background(Color(0xFFC68958)).align(Alignment.Center).size(300.dp))
        Box(Modifier.background(Color.White).align(Alignment.BottomCenter).size(250.dp))
        Box(Modifier.background(Color(0xFFC68958).copy(alpha = 0.5f)).align(Alignment.Center).size(300.dp))
    }
}
```